v1/linkml-schemas/person.yaml

```diff
@@ -5,7 +5,6 @@ description: >-
   A real name for a user, composed of an ordered list of given names and family names (etc)
 prefixes:
   linkml: https://w3id.org/linkml/
-  schema: https://schema.org/
```

Comment:
We're using the `schema` prefix eg in `birthDate` so I'm surprised this works

Reply:
Yeah I was surprised but when I have it in the file I get the following when I build: `WARNING:Namespaces:schema namespace is already mapped to http://schema.org/ - Mapping to https://schema.org/ ignored`

Reply:
I can't figure that out at all, and we may need to come back to it when we look at JSON-LD context creation, but fine then :-)